os/fs/driver/mtd/smart.c

```diff
@@ -3317,14 +3386,15 @@ static uint16_t smart_findfreephyssector(FAR struct smart_struct_s *dev, uint8_t
 	uint32_t readaddr;
 	struct smart_sect_header_s header;
 	int ret;
-
+	uint8_t   *sector_buff;
+	int i;
+	bool bitflipped;
 	/* Determine which erase block we should allocate the new
 	 * sector from. This is based on the number of free sectors
 	 * available in each erase block. */
 
-#ifdef CONFIG_MTD_SMART_WEAR_LEVEL
 retry:
-#endif
+	bitflipped = FALSE;
```

Comment:
This should be added in internal git too.

Reply:
Okay